services/gitdiff/gitdiff_test.go

```diff
@@ -524,6 +524,7 @@ index 0000000..6bb8f39
 	if err != nil {
 		t.Errorf("ParsePatch failed: %s", err)
 	}
+	println(result)
```

Comment:
?

Reply:
I just adopted to the surrounding code which does these prints multiple times.

Reply:
I've removed all 4 cases of `println` in that file. I don't know where it is even defined. I suspect this is something of a testing "global" maybe?

Reply:
It is a Go builtin for bootstrapping/compiler dev, thanks for removing. It is useful only in places where `fmt` can't be imported.

Reply:
Thanks, this was essentially un-googleable, thanks. 

There is the `forbidigo` linter for `fmt.Println` etc, but this is not included there so, I've requested its addition in https://github.com/ashanbrown/forbidigo/issues/32.

Reply:
Actually, `forbidigo` has them already, so this rule would be suitable for us to enable already.